packages/atomic/tsconfig.json

```diff
@@ -1,32 +1,10 @@
 {
-  "extends": "../../tsconfig.json",
-  "compilerOptions": {
-    "allowSyntheticDefaultImports": true,
-    "declaration": false,
-    "experimentalDecorators": true,
-    "lib": ["dom", "ES2023", "ESNext.Collection"],
-    "moduleResolution": "Bundler",
-    "module": "ES2022",
-    "target": "ES2021",
-    "resolveJsonModule": true,
-    "useDefineForClassFields": false,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "jsx": "react",
-    "jsxFactory": "h"
-  },
-  "references": [
-    {
-      "path": "./tsconfig.storybook.json"
-    }
-  ],
-  "include": ["src", "types/jsx.d.ts"],
+  "extends": "./tsconfig.stencil.json",
   "exclude": [
     "node_modules",
     "src/external-builds",
     "**/*.stories.tsx",
     "**/*.stories.ts",
-    "**/*.stories.js",
-    "**/e2e/**/*"
+    "**/*.stories.js"
```

Comment:
Difference between that "exclude" field and the one is tsconfig.stencil.json is this one doesn't have `"**/e2e/**/*"`